content/03.methods.md

```diff
@@ -99,15 +103,18 @@ We filtered outputs from both callers on the "PASS" filter, and annotated using
 
 ### Somatic Copy Number Variant Calling
 
-We used Control-FREEC [@doi:10/ckt4vz; @doi:10/c6bcps] v8.7 and CNVkit [@doi:10.1371/journal.pcbi.1004873] v0.9.3 for copy number variant calls. 
-CNVkit was run using default parameters on human genome reference hg38 and using the batch command for tumor-normal pairs rather than a panel of normals.
+We used Control-FREEC [@doi:10/ckt4vz; @doi:10/c6bcps] v11.6 and CNVkit [@doi:10.1371/journal.pcbi.1004873] v0.9.3 for copy number variant calls. 
+For both algorithms, the `germline_sex_estimate` (described below) was used as input for sample sex and germline variant calls were used as input for B-allele frequency (BAF) estimation. 
+ControlFreeC was run on human genome reference hg38 using the optional parameters of a 0.05 coefficient of variation, ploidy choice of 2-4, and BAF adjustment for tumor-normal pairs.
+Theta2 [doi:10.1093/bioinformatics/btu651] used VarDict germline and somatic calls, filtered on PASS and strongly somatic, to infer tumor purity.
```

Comment:
```suggestion
Theta2 [@doi:10.1093/bioinformatics/btu651] used VarDict germline and somatic calls, filtered on PASS and strongly somatic, to infer tumor purity.
```

Reply:
Why the VarDict calls? It appears to be more permissive than the other callers given the way it was run for this project. Any intuition as to how using another caller would change the results (or perhaps Theta2 requires VarDict)?